spec/controllers/redirect/policy_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Redirect::PolicyController do
+  before do
+    stub_analytics
+  end
+
+  describe '#show' do
+    let(:location_params) { { flow: 'flow', step: 'step', location: 'location', foo: 'bar' } }
+    it 'redirects to policy page' do
+      redirect_url = MarketingSite.security_and_privacy_practices_url
+
+      get :show, params: { **location_params }
```

Comment:
no need to splat if the hash is just the hash

```suggestion
      get :show, params: location_params
```

Reply:
Sounds good - I'll change it in the `contact_controller_spec`, too - since that's where I got it from.